README.md

```diff
@@ -57,4 +57,27 @@ If you are on call or asked to fix an immediate issue, check out our doc on
 [how support works](/playbooks/support.md#readme) and reference our
 [support wiki](https://github.com/artsy/potential/wiki) 🔒 for up-to-date playbooks on how to solve issues.
 
-<a rel="license" href="https://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" />
+<a rel="license" href="https://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a>
+
+## About Artsy
+
+<a href="https://www.artsy.net/">
+  <img align="left" src="https://avatars2.githubusercontent.com/u/546231?s=200&v=4"/>
+</a>
+
+This project is supported by [Artsy][footer_website], a marketplace for buying
+and selling art online. Because one of our most beloved [Engineering
+Principles][footer_principles] is being [Open Source by Default][footer_open],
+we share a lot of how we work with the open source community. You can learn more
```

Comment:
I stumbled a little when reading this sentence, maybe bc "how we work" is ambiguous with respect to whether it refers to what precedes, or follows it.

Maybe…

> Because [...], we share a lot of the details of our work with the open source community.

Not sure that that's any better, or closer to your meaning. Just my 2¢...